### ISSUE TITLE
Add screencap to README

### ISSUE THREAD

**Issue by edvald** [garden-io/garden#155]
It'd be nice to have a quick screencap showing basic usage on the README, helps to explain at a glance what Garden is and does.

**Comment by ellenkorbes** [comment]
I'll just leave this here before I forget about it: https://asciinema.org/

**Comment by edvald** [comment]
👍, just one feature you missed (which tells me it needs better docs): `garden call hello-container/hello`. Does away with the curl and maybe even the status command (but you might want to show it anyway, not sure).

This feels self explanatory to me, especially in the context, but I may be biased :þ

**Comment by eysi09** [comment]
I don't have the tmux issue but the recording looks bad though do the way the logger updates the screen: https://asciinema.org/a/Jt3uFSRyqbw8gbBQ3WBZmIZKv

**Comment by ellenkorbes** [comment]
Hmmm. On my end I'm not having *that* issue https://asciinema.org/a/MX6kSfvVYytIo7OQk7aFM3ZwI 🤔

**Comment by ellenkorbes** [comment]
What do you guys think? https://asciinema.org/a/peIaRYTt3Pt6JXteUgsPfIr58

**Comment by edvald** [comment]
I like the flow of it, we'll constructed, just a couple of thoughts:

1 - it's maybe not immediately obvious what's happening when you switch terminals, so it might be good to type comments before/after switching.

2 - the wait times are quite long (considerably longer than on my machine, even before I got the new one...). It might make sense to edit a bit for effect.

3 - I'm not sure if it's possible to get our lovely banner image to display when running `garden dev`, which I think only works on iTerm anyway, but in any case the welcome message could be improved when it doesn't work. Not an issue with the demo as such, just something I notice.

4 - maybe change the terminal setting to omit the git branch while recording? Also helps with the line length.

**Comment by ellenkorbes** [comment]
@edvald:

1. Good point.

2. Yeah, performance here with minikube has been really, really sub-par. I guess I'll just record it with one of your machines next week; it's less trouble than learning to edit whatever crazy format asciinema uses. 

3. I'll file a separate issue for that.

4. Good idea.

**Comment by edvald** [comment]
Just came across this: https://news.ycombinator.com/item?id=17449810

**Comment by ellenkorbes** [comment]
### Instructions so someone else can do the recording instead of me:

#### Preparation

1. You need `tmux` and `asciinema` installed.
2. Create three windows on `tmux` (I'll be calling them terminals 0, 1, and 2).
3. Terminals 0 and 1 should be at `garden/examples/hello-world`. 
4. Terminal 2 should have `garden/examples/hello-world/services/hello-function/index.js` open.
5. Detach your `tmux` session.
6. To start recording, do `asciinema rec -c "tmux attach -t session-name"` or, if it's the only session you have running, simply `asciinema rec -c "tmux a"`

#### Recording

```
START AT TERMINAL #0
# Hello! This is an example project for the garden framework.
# It contains two modules: a container and a serverless function.
ls services
# Let's use garden to test and deploy them!
garden dev
SWITCH TO TERMINAL #1
# Let's see what our project looks like.
garden get status
OPTIONAL: SCROLL UP AND DOWN A BIT
# And let's test that it works.
garden call hello-container/hello
# You can access it however you want! E.g.
curl http://hello-container.hello-world.192.168.99.100.nip.io:80/hello
# Now let's change something. Then we'll move back to `garden dev` and see it re-test and re-deploy.
SWITCH TO TERMINAL #2
CHANGE services/hello-function/index.js from hello("function") to hello("pizza")
SWITCH TO TERMINAL #0 AND WAIT FOR TEST TO FAIL
# Oops! Looks like it wasn't expecting pizza!
SWITCH TO TERMINAL #1
# The deployment works as expected though:
garden call hello-container/hello
# We hope you've enjoyed this little demo! For more go to github.com/garden-io/garden!
DETACH SESSION AND PRESS ENTER TO PUBLISH
```

**Comment by eysi09** [comment]
Fixed in #233
